src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -59,6 +59,10 @@ public void setTestVerbosity(){
     public static final String publicTestDir = new File(gatkDirectory, publicTestDirRelative).getAbsolutePath() + "/";
     public static final String publicTestDirRoot = publicTestDir.replace(publicTestDirRelative, "");
 
+    public static final String GCS_b37_REFERENCE_2BIT = "gs://hellbender/test/resources/benchmark/human_g1k_v37.2bit";
+    public static final String GCS_b37_CHR20_21_REFERENCE_2BIT = "gs://hellbender/test/resources/human_g1k_v37.20.21.2bit";
+    public static final String GCS_b37_CHR17_1M_REFERENCE_2BIT = "gs://hellbender/test/resources/human_g1k_v37.chr17_1Mb.2bit";
```

Comment:
Is there a declaration in `BaseTest` for our standard GCS bucket for testing (if not, can you please create one)? Also, I thought at some point we had switched to `gs://hellbender`